### PR TITLE
fix: logo icon briefly disappears and reappears during theme change

### PR DESCRIPTION
**Pull request by JasonFengGit**
Fix https://github.com/electron/website/issues/539.

Current Behavior: the logo still blinks sometimes when changing theme, but the layout would not be messed up. This is the same behavior as https://docusaurus.io/docs

Built and tested on my local environment (windows, chrome) with different window sizes.

Related: https://github.com/facebook/docusaurus/issues/8188

### CODE REVIEW

File: src/css/custom.scss [hunk 1]
```diff
@@ -250,3 +250,8 @@
 .table-of-contents__link--active {
   font-weight: var(--ifm-font-weight-bold);
 }
+
+// avoid layout change during theme change
+.navbar__logo {
+  width: 2rem;
+}
```
Comment (BlackHole1) [inline src/css/custom.scss:257]:
```suggestion
}

```

File: src/css/custom.scss [hunk 2]
```diff
@@ -250,3 +250,8 @@
 .table-of-contents__link--active {
   font-weight: var(--ifm-font-weight-bold);
 }
+
+// avoid layout change during theme change
```
Comment (BlackHole1) [inline src/css/custom.scss:254]:
```suggestion
// fix https://github.com/electron/website/issues/539
```

**Review by BlackHole1 — CHANGES_REQUESTED** [review]

**Comment by JasonFengGit** [comment]
Hi @BlackHole1, I have applied the requested changes. Thank you so much for the suggestions!

**Review by BlackHole1 — APPROVED** [review]
LGTM

**Review by dsanders11 — APPROVED** [review]

**Pull request merged**